src/components/Table.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface TableProps {
+  adjacentNodes: [string, number][];
+}
+
+const Table: React.FC<TableProps> = ({ adjacentNodes }) => {
+  return (
+    <div>
+      <table style={{ borderCollapse: 'collapse', width: '100%' }}>
+        <thead>
+          <tr>
+            <th style={{ border: '1px solid white' }}>Related Repository</th>
+            <th style={{ border: '1px solid white' }}>Relation value</th>
+          </tr>
+        </thead>
+        <tbody>
+          {adjacentNodes.map((node, index) => (
+            <tr key={index}>
+              <td style={{ border: '1px solid white', textAlign: 'center', verticalAlign: 'middle' }}>{node[0]}</td>
+              <td style={{ border: '1px solid white', textAlign: 'center', verticalAlign: 'middle' }}>{node[1]}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
```

Comment:
We need to change the table style based on Lam's desgin.
<img width="265" alt="image" src="https://github.com/user-attachments/assets/069a8a89-510b-43e9-9504-ef6c78d644fe">
1. Remove the border color
2. text in the first column should be left align
3. We need to add avatars (This is important because users need avatars to matching highlighted nodes in graph and names in table)
4. We need to handle the situation when there are too many to show. Add a <div> as a table container and set its max height and overflow.
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/17b9a4d1-f798-408c-bd02-fa10cf6e5939">
5. You can have a variable to hold the same style instead of setting it repeatedly.
6. We also need margins for the tables